src/lib/Form/Data/FieldDefinitionData.php

```diff
@@ -18,6 +18,8 @@
  */
 class FieldDefinitionData extends FieldDefinitionUpdateStruct
 {
+    public bool $enabled;
```

Comment:
Data/information architecture: so does this mean that now all Field Definitions can be enabled? I thought this was specific to Meta Fields.